### DISCUSSION TITLE
Out of date custom mailer docs

### DISCUSSION THREAD

**Discussion by mdodell** [janko/rodauth-rails#318] [Q&A]
I noticed that [these docs](https://github.com/janko/rodauth-rails/wiki/Custom-Mailer-Worker) were fairly out of date - Sidekiq now calls workers 'jobs'. There is also a `send_email` function which I believe is where we should be sending emails from.

I'm trying to set up Rodauth with a custom mailing system using Sendgrid or Postmark. Could these docs be updated as that feels like a common use case?

**Comment by janko** [comment]
The wiki is publicly editable, so feel free to correct any outdated terminology/API.

As for `send_email`, I'm not sure how it could be used here. It receives the result of `create_*_email` methods as the argument, but not sure what that object could be. Unless I'm missing something, overriding `send_*_email` seems correct here.
